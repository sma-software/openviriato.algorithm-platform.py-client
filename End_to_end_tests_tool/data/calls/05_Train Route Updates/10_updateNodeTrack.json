{
    "restResource": "/trains/{trainId}/train-path-nodes/{trainPathNodeId}:update-node-track",
    "httpMethod": "PUT",
    "urlParameters": [ "=sepTrain:id=", "=sepTrain:trainPathNodes[2].id=" ],
    "requestBody": {
        "nodeTrackId": "162"
    },
    "CSharpMethodName": "UpdateNodeTrack(trainID, trainPathNodeID, nodeTrackID)",
    "CSharpMethodParameterTypes": [ "System.Int64", "System.Int64", "System.Nullable{System.Int64}" ],
    "subcategory": "Update Node Track"
}
