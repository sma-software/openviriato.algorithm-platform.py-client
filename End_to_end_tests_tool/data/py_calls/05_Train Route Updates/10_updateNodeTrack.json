{
    "PythonParameterMapping": {
        "train_id" : "$.urlParameters.[0]",
        "train_path_node_id" : "$.urlParameters.[1]",
        "node_track_id" : "$.requestBody.nodeTrackId"
    }
}