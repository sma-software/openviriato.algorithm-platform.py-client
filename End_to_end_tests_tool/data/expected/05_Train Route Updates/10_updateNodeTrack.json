{
  "id": 6430,
  "trainPathNodes": [
    {
      "id": 5338,
      "sectionTrackId": null,
      "nodeId": 321,
      "nodeTrackId": 322,
      "formationId": 4108,
      "arrivalTime": "2003-05-01T06:00:00",
      "departureTime": "2003-05-01T06:00:00",
      "minimumRunTime": null,
      "minimumStopTime": "P0D",
      "stopStatus": "commercialStop",
      "sequenceNumber": 0
    },
    {
      "id": 5702,
      "sectionTrackId": 888,
      "nodeId": 281,
      "nodeTrackId": null,
      "formationId": 4108,
      "arrivalTime": "2003-05-01T06:02:42",
      "departureTime": "2003-05-01T06:02:42",
      "minimumRunTime": "PT2M42S",
      "minimumStopTime": "P0D",
      "stopStatus": "passing",
      "sequenceNumber": 1
    },
    {
      "id": 6066,
      "sectionTrackId": 887,
      "nodeId": 161,
      "nodeTrackId": 162,
      "formationId": null,
      "arrivalTime": "2003-05-01T06:03:54",
      "departureTime": "2003-05-01T06:03:54",
      "minimumRunTime": "PT1M12S",
      "minimumStopTime": "P0D",
      "stopStatus": "commercialStop",
      "sequenceNumber": 2
    }
  ],
  "debugString": "FV_9_J03 tt_(SZ)"
}